src/_data/catalog/widgets.json

```diff
@@ -1166,6 +1342,39 @@
     "link": "https://api.flutter.dev/flutter/widgets/MergeSemantics-class.html",
     "image": "<img alt='' src='/assets/images/docs/catalog-widget-placeholder.png'>"
   },
+  {
+    "name": "Navigation bar",
+    "description": "Navigation bars offer a persistent and convenient way to switch between primary destinations in an apps.",
+    "categories": [],
+    "subcategories": [
+      "Navigation"
+    ],
+    "link": "https://api.flutter.dev/flutter/material/NavigationBar-class.html",
+    "image": "<img alt='' src='/assets/images/docs/widget-catalog/material-3-navigation-bar.png'>",
+    "hoverimage": "<img alt='' src='/assets/images/docs/widget-catalog/material-3-bubbles.png'>"
+  },
+  {
+    "name": "Navigation drawer",
+    "description": "Navigation drawers provide ergonomic access to destinations in an app.",
```

Comment:
**issues:**
- Make consistent with other descriptions.
- Remove the circular definition.
- Provide more context about the widget.
```suggestion
    "description": "Container that slides from the side of the app to navigate to other sections in an app.",
```